news/2020-04-10-osu-spotlights-new-team-and-league-system.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "osu! Spotlights: New Team and League System"
```

Comment:
could be mistakenly read as "new team system" which sounds like teams of players instead of the spotlights team